### ISSUE TITLE
Are these scripts still working?

### ISSUE THREAD

**Issue by emsariel** [Matthewspear/OFHabitRPG#16]
I've been using this for about a week, now, and ... I'm not sure that the integration is still working.  The script runs fine, and without error that I can see.  However, a refresh in Habitica a few minutes after the script has completed doesn't show any change in XP, mana, or drops.  I could swear it worked early on, but I'm thinking I may have just completed a task in Habitica itself around the same time.

@Matthewspear - it sounded like you have not been using Habitica recently.  Do you know that the scripts, properly configured, still work?

If it's not still working, or we're not sure, then I'll work on the logging first, and add some error handling to catch complaints from the Habitica API.

**Comment by matthewspear** [comment]
Sadly I've moved away from using both OmniFocus and Habitica, might get back into it at some point and people found this script an interesting idea / useful so decided to keep it around.

On the recent update I did a few tests, checking the habits that triggered and the daily count available when you edit them.

It may be also worth seeing if there is a way of displaying errors that occur so that the user is notified and can re-try submitting them.
